File: Sort/BubbleSort/BubbleSort.java
public class BubbleSort
{
    public static int[] bubbleSort(int arr[]){
        int temp;
		for(int j=0; j < arr.length - 1; j++){
		    for(int i=1; i < arr.length - j; i++){
    		    if(arr[i - 1] > arr[i]){
    		        temp = arr[i - 1];
    		        arr[i - 1] = arr[i];
    		        arr[i] = temp;
    		    }
    		}
		}
		return arr;
    }
	public static void main(String[] args) {
		int[] arr = {3, 6, 2, 9, 8, 7, 1, 4, 5};
		bubbleSort(arr);
		for (int x : arr){
		    System.out.println(x);
		}
	}
}

File: Sort/BubbleSort/BubbleSort.py
def bubbleSort(arr):
    for j in range(len(arr) - 1):
        for i in range(1, len(arr) - j):
            if (arr[i - 1] > arr[i]):
                temp = arr[i - 1]
                arr[i - 1] = arr[i]
                arr[i] = temp
    return arr
if __name__ == "__main__":
    arr = [3, 6, 2, 9, 8, 7, 1, 4, 5]
    print(bubbleSort(arr))